lib/ui/platform_dispatcher.dart

```diff
@@ -1290,6 +1291,10 @@ class PlatformConfiguration {
 class ViewConfiguration {
   /// A const constructor for an immutable [ViewConfiguration].
   const ViewConfiguration({
+    this.view,
+    @Deprecated('''
+      Renaming window to view since `FlutterWindow` has been removed from dart::ui.
```

Comment:
1. I don't understand this deprecation message. It should be something actionable in the form of "Use xxxx instead." For example, "Use the `view` argument instead. The `window` argument has been renamed to `view`."
2. If it's a rename, shouldn't we point them to the same value? The two parameter should be exclusive (assert that only one of them is non-null) and be assigned to the same field (`_view`) and both getter should return this field.

Reply:
Yeah, you're right 👍

Reply:
How about 
```
      `FlutterWindow` has been removed from dart::ui, and is illogical semantically
       use the `view` property instead.
```

Reply:
In my opinion the explanation is unnecessary here. Just tell the user how to change it. If you really want to explain it, add a link to the design doc, or to this PR with an expanded paragraph of description.

Reply:
> For example, "Use the `view` argument instead. The `window` argument has been renamed to `view`."

+1 for this deprecation message

Reply:
Reopening as I don't think the second part was addressed:

> If it's a rename, shouldn't we point them to the same value? The two parameter should be exclusive (assert that only one of them is non-null) and be assigned to the same field (_view) and both getter should return this field.